0000-repo-migration.md

```diff
@@ -0,0 +1,126 @@
+# Paketo Repo Migration Proposal
+
+## Summary
+Below we propose what repositories we will migrate into the paketo-buildpacks org.
+
+## Implementation
+
+#### NodeJS CNB (language family and implementation CNBs)
+
+- node-engine
+- yarn-install
+- npm
+ 
+#### Go CNB (language family and implementation CNBs)
+
+- go-compiler
+- go-mod
+- dep
+
+#### Dotnet Core CNB (language family and implementation CNBs)
+
+- dotnet-core-runtime
+- dotnet-core-aspnet
+- dotnet-core-sdk
+- dotnet-core-build
+- dotnet-core-conf
+- icu
+
+#### PHP CNB (language family and implementation CNBs)
+
+- php-web
+- php-composer
+- php-dist
+
+#### Httpd CNB (language family and implementation CNBs)
+
+- httpd
+
+#### Nginx CNB (language family and implementation CNBs)
+
+-nginx
+
+#### Java CNB (language family and implementation CNBs)
```

Comment:
- `adopt-openjdk`
- `amazon-corretto`
- `apache-tomcat`
- `azul-zulu`
- `azure-application-insights` (Java and NodeJS Implementations)
- `bellsoft-liberica`
- `build-system`
- `debug`
- `dist-zip`
- `eclipse-openj9`
- `encrypt-at-rest`
- `executable-jar`
- `google-stackdriver` (Java and NodeJS Implementations)
- `jmx`
- `procfile`
- `sap-machine`
- `spring-boot`